packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -42,7 +42,7 @@ export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivEle
   defaultActiveKey?: number | string;
   /** Callback to handle tab selection */
   onSelect?: (event: React.MouseEvent<HTMLElement, MouseEvent>, eventKey: number | string) => void;
-  /** @beta Callback to handle tab closing */
+  /** @beta Callback to handle tab closing and adds a basic close button to all tabs. This is overridden by the Tab actions property. */
```

Comment:
nit. lower case tab
Based on the discusion above in Tab.tsx, this description may need to be updated.

Reply:
I'll update it for now. Can always remove/edit it later if we end up taking it out.